Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN apt-get install -y \
     libsdl1.2debian \
     libsdl1.2-dev
 
-WORKDIR /ros2_ws
+WORKDIR /ros2_dependencies_ws
```

Comment:
I don't think this is the repo that @mkhansen-intel was referring to. I think this should stay as `ros2_ws` since that is what OSRF calls it in all their instructions.

The one Matt was referring to is navstack_dependencies_ws.

I'd prefer we back out this one commit and stick with ros2_ws for this one.

The other one `navstack_dependencies_ws`, I have mixed feelings on changing, so I don't care if you fix it or not in this PR.

Reply:
Woops. I missed that this was the top-level directory in the dockerfile. 

My comment applies more to the build and install scripts

For the dockerfile, this directory contains dependencies and the navigation2_ws, so I don't think ros2_dependencies_ws is really correct either.